src/collections/_documentation/guides/integrate-frontend/configure-scms.md

```diff
@@ -24,13 +24,13 @@ In this part, you will:
 
     ![Add project repository]({% asset guides/integrate-frontend/configure-scms-01.png @path %})
 
-GitHub should now be enabled for all projects under your Sentry organization.
+GitHub should now enable and available for all projects under your Sentry organization.
```

Comment:
missing a "be" ? "...be enabled and available "

Reply:
yep. done